certs_to_crlite.py

```diff
@@ -0,0 +1,49 @@
+import censys.certificates
+from decouple import config
+from filter_cascade import FilterCascade
+import sys
+
+CENSYS_API_UID = config("CENSYS_API_UID", None)
+CENSYS_API_SECRET = config("CENSYS_API_SECRET", None)
+
+if not CENSYS_API_UID or not CENSYS_API_SECRET:
+    print "Must set CENSYS_API_UID and CENSYS_API_SECRET"
+    sys.exit(1)
+
+certificates = censys.certificates.CensysCertificates(
+    CENSYS_API_UID, CENSYS_API_SECRET
+)
+
+MLBF_FILENAME = 'moz-crlite-mlbf'
+
+a = []
+b = []
+
+certs_list = certificates.search(
+    'parsed.issuer.organization.raw: "DigiCert Inc"'
+)
+
+for idx, c in enumerate(certs_list):
+    print "Iteration %i" % idx
+    if idx == 500:
+        break
+    # let's say every third cert should *not* be in the filter
+    shouldBeRevoked = (idx % 3 == 1)
```

Comment:
I'm assuming you're just using the same means of omitting certain entries so we end up with a filter that does not contain the full set of entries (as I did with the dictionary example)? For actual data, we'd want to use the revocation state of each certificate to determine inclusion in the filter.

Reply:
Right, this is just to have some entries in the "revoke" list and some in the "non-revoked" list.